php/ext/google/protobuf/protobuf.h

```diff
@@ -127,7 +127,7 @@ ZEND_BEGIN_ARG_INFO_EX(arginfo_setter, 0, 0, 1)
   ZEND_ARG_INFO(0, value)
 ZEND_END_ARG_INFO()
 
-#define PHP_PROTOBUF_VERSION "3.21.10"
```

Comment:
I wonder why this wasn't bumped to 11...

Reply:
I was releasing 3.21.12

Reply:
likely a merge error on 3.21.11

Reply:
Yea, but it should have been 11 in main, not 10.  It looks like we didn't merge back to main for 21.11?

Reply:
https://github.com/protocolbuffers/protobuf/pull/11210 happened

Reply:
https://github.com/protocolbuffers/protobuf/commit/032c9c8ada0587ffc41d5e635d9bb3ce2be5c16f rolled it back